### PR TITLE
Bug 1967658: improve failure alert for copied CSV

### PR DESCRIPTION
**Pull request by rhamilto**
After:
If CSV is copied:
Screen Shot 2021-06-14 at 10 28 17 AM
Otherwise:
Screen Shot 2021-06-14 at 11 45 32 AM

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx [hunk 1]
```diff
@@ -987,8 +988,27 @@ export const ClusterServiceVersionDetails: React.FC<ClusterServiceVersionDetails
                   isInline
                   className="co-alert"
                   variant="danger"
-                  title={`${status.phase}: ${status.message}`}
-                />
+                  title={
+                    status.reason === CSVConditionReason.CSVReasonCopied
+                      ? t('olm~Operator failed')
+                      : `${status.phase}: ${status.message}`
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx:994]: I would always show "Operator failed" here and then show `status.message` in the alert body below when reason isn't copied.

**Review by spadgett — COMMENTED** [review]
Thanks @rhamilto

File: frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx (hunk 1 above)
Comment (rhamilto) [reply frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx]: Thanks, @spadgett.  Updated.

**Review by spadgett — APPROVED** [review]
/lgtm

👍

**Pull request merged**